.pre-commit-config.yaml

```diff
@@ -1,10 +0,0 @@
-repos:
```

Comment:
I think we can keep this file, I often forget to `make helm-docs` manually, and it will help me do this.

Reply:
Sure let's keep it